src/fluree/db/query/exec/update.cljc

```diff
@@ -146,15 +146,15 @@
               [s-mch p-mch o-mch] (where/assign-matched-values triple solution nil)
 
               s-cmp  (if (string? (::where/val s-mch))
-                       (assoc s-mch ::where/val (<? (dbproto/-subid db (::where/val s-mch) true)))
+                       (assoc s-mch ::where/val (<? (dbproto/-subid db (::where/val s-mch) {:expand? false})))
```

Comment:
i'm assuming that using match-value instead of setting `::where/val` directly will happen in one of the other prs.

Reply:
Yeah, this came first.